src/client/pythonEnvironments/base/info/index.ts

```diff
@@ -28,6 +28,19 @@ export enum PythonEnvKind {
     OtherVirtual = 'virt-other'
 }
 
+export function getGlobalEnvKinds() : PythonEnvKind[] {
+    return [
+        PythonEnvKind.System,
+        PythonEnvKind.MacDefault,
+        PythonEnvKind.WindowsStore,
+        PythonEnvKind.Pyenv,
+        PythonEnvKind.CondaBase,
+        PythonEnvKind.Poetry,
+        PythonEnvKind.Custom,
+        PythonEnvKind.OtherGlobal,
+    ];
+}
```

Comment:
We could lean on internal information here to simplify things and benefit from DRY (i.e. future-proof).  Something like this:

```suggestion
function isGlobalEnvKind(kind: PythonEnvKind): boolean {
    return kind.startsWith('global-');
}

export function getGlobalEnvKinds() : PythonEnvKind[] {
    const kinds = getNames(PythonEnvKind) as PythonEnvKind[];
    return kinds.filter(isGlobalEnvKind);
}
```

Where `getNames()` comes from src/client/common/utils/enum.ts.

Reply:
@ericsnowcurrently Can workspace virtual environments be of type `PythonEnvKind.Custom` as well? Why do we have only a global custom type?

Reply:
A "custom" environment is a Python build (e.g. "./configure & make install") that someone did manually.  Virtual environments are wrappers around global Python installs, so "custom" virtual environments do not exist.